project/src/styles/not-found-screen.css

```
@@ -0,0 +1,26 @@
+.error-container {
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.error-content {
+  text-align: center;
+}
+
+.error-code {
+  margin-bottom: 0px;
```

Для нуля не следует указывать единицы измерения
```suggestion
  margin-bottom: 0;
```